src/tentaclio/__init__.py

```diff
@@ -64,6 +64,7 @@
 # Db registry
 DB_REGISTRY.register("postgresql", PostgresClient)
 DB_REGISTRY.register("awsathena+rest", AthenaClient)
+DB_REGISTRY.register("databricks", DatabricksClient)
```

Comment:
should we use `databricks+pyodbc` as the scheme to align completely with sqlalchemy?